src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -653,37 +659,33 @@ export class BaseEditCourseForm extends React.Component {
               extraInput={{ onInvalid: this.openCollapsible }}
               disabled={courseInReview}
             />
-            {entitlement && (
-              <React.Fragment>
-                <Field
-                  name="mode"
-                  component={RenderSelectField}
-                  label={
-                    <FieldLabel
-                      id="mode.label"
-                      text="Enrollment track"
-                      helpText={enrollmentHelp}
-                    />
-                  }
-                  extraInput={{ onInvalid: this.openCollapsible }}
-                  options={this.getEnrollmentTrackOptions()}
-                  disabled={courseInReview}
-                />
-                <Field
-                  name="price"
-                  component={RenderInputTextField}
-                  type="number"
-                  label={<FieldLabel text="Price" required requiredForSubmit />}
-                  extraInput={{
-                    onInvalid: this.openCollapsible,
-                    min: 0.01,
-                    step: 0.01,
-                  }}
-                  disabled={courseInReview}
-                  required={isSubmittingForReview}
+            <Field
+              name="mode"
+              component={RenderSelectField}
+              label={
+                <FieldLabel
+                  id="mode.label"
+                  text="Enrollment track"
+                  helpText={enrollmentHelp}
                 />
-              </React.Fragment>
-            )}
+              }
+              extraInput={{ onInvalid: this.openCollapsible }}
+              options={this.getEnrollmentTrackOptions()}
+              disabled={courseInReview || (!!entitlement && !!entitlement.sku)}
+            />
+            {currentValues && ENTITLEMENT_TRACKS.includes(currentValues.mode) && <Field
```

Comment:
If this field doesn't exist because of audit mode, when we PATCH, do we serialize anything weird for price?

Reply:
On creation, we will send `price: undefined` from the frontend and the backend gives a default value of 0.00 if price does not exist. So in the end, we will create a draft entitlement with mode Audit and price 0.00. Any future saves, will send price = 0.00.

So it's documented, if you start verified and set a price of say $50 and then later switch to Audit. The Audit saves will still be sent with the $50 because that was the most recent saved form value so it still gets sent. This will update the draft entitlement to be Audit with a price, but we still won't create an official entitlement because the mode is not valid. The silver lining of this is that if the user is switching between audit and verified, we are "saving" their last known price value.